### PR TITLE
Khalid/added 18+ underage sign to footer

### PR DESCRIPTION
**Pull request by Khalidxx**
Also, a small change to the URL for requesting style files

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "babel-preset-stage-0": "^6.5.0",
     "concurrently": "^3.3.0",
     "connect": "^3.6.1",
-    "grunt": "^1.0.1",
```
Comment (cakasuma) [inline package.json]: can do this in seperate PR

File: src/index.scss [hunk 2]
```diff
@@ -140,6 +143,10 @@ body {
         > * {
             margin-right: 20px;
         }
+        @media screen and (max-width: 800px)  {
```
Comment (cakasuma) [inline src/index.scss]: i think we can combine these two media queries since it is the same pixel

File: package-lock.json [hunk 3]
```diff
@@ -1892,10 +1892,10 @@
       "integrity": "sha1-DQcLTQQ6W+ozovGkDi7bPZpMz3c=",
```
Comment (OskarAhl) [inline package-lock.json]: Can we move the `package-lock.json` changes to a separate card?

Comment (Khalidxx) [reply package-lock.json]: Alright, resolved

Comment (OskarAhl) [reply package-lock.json]: Thanks 👍

**Pull request merged**